src/utils/__tests__/scroll-test.ts

```diff
@@ -27,34 +33,46 @@ describe('scrollToLocation', () => {
             container.scrollLeft = left || 0;
             container.scrollTop = top || 0;
         });
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+        // @ts-ignore
```

Comment:
I think we accomplish this same feat without the use of `ts-ignore` in other files.

Reply:
It's literally used above on L28 :) https://github.com/box/box-annotations/pull/538#diff-dde97b94dedd5c5c622549ce1ec1524eR28

Reply:
Yes, for a different method. Here's an existing [example](https://github.com/box/box-annotations/blob/master/src/common/useAutoScroll/__tests__/useAutoScroll-test.tsx#L17) of how to mock getBoundingClientRect.